### PR TITLE
ADD Dashbord

### PR DESCRIPTION
**Pull request by bodiatym**

### CODE REVIEW

File: app/controllers/menus_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class MenusController < ApplicationController
+  before_action :set_menu
+
+  def show; end
+
+  def set_menu
```
Comment (anastasiabalaban) [inline app/controllers/menus_controller.rb:8]: should be private

File: app/views/menus/show.html.haml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+%section.container
+%h1= t('.menu', day: 'Test Day')
```
Comment (anastasiabalaban) [inline app/views/menus/show.html.haml:2]: emplty line

File: app/views/pages/dashboard.html.haml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+%section.container
+%h1= "Dashboard"
+%p
+  %li= link_to t('.menu',day: 'Test Day'), menu_path(1) , class: "btn btn-primary btn-sm mt-10"
```
Comment (anastasiabalaban) [inline app/views/pages/dashboard.html.haml:4]: menu_path(1) ???

please review your spaces

File: test/models/menu_test.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class MenuTest < ActiveSupport::TestCase
```
Comment (anastasiabalaban) [inline test/models/menu_test.rb:5]: as I remember you use rspec

File: app/views/pages/dashboard.html.haml (hunk 3 above)
Comment (bodiatym) [reply app/views/pages/dashboard.html.haml:4]: It's just a test because I don't know how to do menus properly

**Pull request closed without merging**